src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -266,7 +253,7 @@ public static void closeMessageLighting() {
                 WEB_DRIVER.findElement(By.id("tryLexDialogX")).click();
             }
         } catch (NoSuchElementException e) {
-            e.printStackTrace();
+            System.out.println("Exception");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)